rustport/cppshim/include/bindings.hpp

```diff
@@ -0,0 +1,40 @@
+#include "bindings.h"
+
+#include <iostream>
+#include <type_traits>
+#include <utility>
+
+namespace rust {
+
+/// @brief Asynchronously invoke f on the rust-backed executor.
```

Comment:
```suggestion
/// @brief Asynchronously invokes f on the rust-backed executor.
```

Descriptive, not prescriptive.